crypto/x509/x509_lcl.h

```diff
@@ -16,7 +16,7 @@
 struct X509_VERIFY_PARAM_st {
     char *name;
     time_t check_time;          /* Time to use */
-    unsigned long inh_flags;    /* Inheritance flags */
+    uint32_t inh_flags;         /* Inheritance flags */
```

Comment:
Because "long" is a useful type in an ABI (no formal width defined).  So making it uint32 is better and since the structure is opaque, I can make the change all the way back to the structure itself.